### PR TITLE
fix(docs): update readme and docs for registry

### PR DESCRIPTION
**Pull request by omercnet**
adding registry docs and correcting package provider in readme

related to https://github.com/pulumi/registry/pull/2992

### CODE REVIEW

File: docs/_index.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+---
+title: Vercel
+meta_desc: Provides an overview of the Vercel Provider for Pulumi.
+layout: overview
```
Comment (ringods) [inline docs/_index.md:4]: Value for `layout` must be `package`.

File: docs/installation-configuration.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
+---
+title: Vercel Installation & Configuration
+meta_desc: Information on how to install the Vercel provider.
+layout: installation
```
Comment (ringods) [inline docs/installation-configuration.md:4]: Value for `layout` must be `package`.

**Review by ringods — CHANGES_REQUESTED** [review]

File: docs/_index.md (hunk 1 above)
Comment (omercnet) [reply docs/_index.md:4]: This was taken from the example you sent..

**Comment by omercnet** [comment]
@ringods fixed your requests

**Review by ringods — APPROVED** [review]

**Pull request merged**